.github/actions/composite/setupNode/action.yml

```diff
@@ -9,7 +9,15 @@ runs:
         node-version-file: '.nvmrc'
         cache: npm
 
+    - id: cache-node-modules
+      if: github.run_attempt == '1'
```

Comment:
[Question 1 and 2 here](https://github.com/Expensify/App/pull/20436#issuecomment-1583074039) is me wondering if we can just remove this line and the `github.run_attempt != '1'` below.

Reply:
The purpose of this code is to make it so that we will not use the cache if a workflow is ever _manually retried_.

My reasoning is because if we're manually retrying a workflow, then something must have gone wrong, and doing a clean install is a good idea because it removes the cache from the equation. I could be convinced otherwise but it seems like a good way to make it so deployers have to think about caches less

Reply:
Ah I see, I think personally I'd rather have it always on and then just know that we can delete them via https://github.com/Expensify/App/actions/caches

(This is how it works in Travis too)

However, I do not feel strongly enough to block. If we do keep this code, let's add a comment explaining this is how it works.

Reply:
Thoughts on this one @roryabraham ?

Reply:
Ok, I removed the `run_attempt` logic from here and from the pod caching as well